marquez/utils/request_utils.py

```diff
@@ -0,0 +1,96 @@
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import collections
+import json
+import logging
+import urllib
+from http import HTTPStatus
+
+import requests
+from marquez.constants import NO_CONTENT_RESPONSE, NOT_FOUND
+
+
+def put_request(marquez_host, path, path_args=[], args=None, **kwargs):
+    path = _compose_path(path, path_args)
+    full_path = "http://{0}/{1}".format(marquez_host, path)
+    if args:
+        logging.info("args are: " + json.dumps(args))
```

Comment:
do we need to log the parameters? this will potentially show up in anything which relies on the client (like Airflow DAGs)

Reply:
Removed